File: app/src/main/java/com/example/aac/data/activity/TestDataActivity.java
package com.example.aac.data.activity;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.support.v7.widget.Toolbar;
import android.widget.TextView;

import com.aac.expansion.data.AacDataActivity;
import com.aac.module.pres.RequiresPresenter;
import com.center.toolbar.TitleToolbar;
import com.example.aac.R;

/**
 * Created by yangc on 2017/8/14.
 * E-Mail:yangchaojiang@outlook.com
 * Deprecated:
 */

@RequiresPresenter(TesDataPresenter.class)
public class TestDataActivity extends AacDataActivity<TesDataPresenter, String> {
    TextView te;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        te = findViewById(R.id.textView2);
        initLoadHelper(findViewById(R.id.root));
        getSupportActionBar().setTitle("测试");
    }


    @Override
    public void setData(@NonNull String data) {
        te.setText(data);
    }

    @Override
    public int getContentLayoutId() {
        return R.layout.test_data1_views;
    }
}

File: kotlintest/src/main/java/com/example/kotlin/activity/TestDataActivity.kt
package com.example.kotlin.activity

import com.aac.expansion.data.AacDataActivity
import com.aac.module.pres.RequiresPresenter
import com.example.kotlin.R
import kotlinx.android.synthetic.main.test_data_activity.*
/**
 * Created by yangc on 2017/8/27.
 * E-Mail:yangchaojiang@outlook.com
 * Deprecated:
 */
@RequiresPresenter(TestDataPresenter::class)
public  class TestDataActivity : AacDataActivity<TestDataPresenter, String>(){

    override fun getContentLayoutId():Int=  R.layout.test_data_activity

    override fun setData(data: String) {
        textView.text=data
    }

    override fun setError(e: Throwable?) {

    }

}